package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/helix-content-proxy",
-  "version": "2.9.7",
+  "version": "2.9.6-test",
```

Comment:
I guess this will be changed back to something without `-test`?

Reply:
yes, semantic release will create a new version.